src/index.js

```diff
@@ -161,6 +161,7 @@ class AlertsPlugin {
         alarm.Properties.Statistic = definition.statistic
       } else {
         alarm.Properties.ExtendedStatistic = definition.statistic
+        alarm.Properties.EvaluateLowSampleCountPercentile = definition.evaluateLowSampleCountPercentile
```

Comment:
I'm not sure whether CFN accepts `undefined`, but we can test

Reply:
I tested and it defaults to 'evaluate'. When you json stringify an undefined value it removes it: 
```
> JSON.stringify({ a: undefined, b: 'asdf' });
'{"b":"asdf"}'
```